music21/repeat.py

```diff
@@ -1082,7 +1082,6 @@ def _groupRepeatBracketIndices(self, streamObj):
                                   <music21.stream.Measure 2 offset=3.0>>]}]
         '''
         groups = []
-        mEnumerated = [x for x in enumerate(streamObj)]
```

Comment:
This might make a change because there is a break inside the `for ... in enumerate(streamObj)`.  Better to cast as a list here, which will set all activeSites -- probably doesn't make a difference, but trying to be most careful

Reply:
yeah I was worried about this one